tests/unit/test_ringcells_reports.py

```diff
@@ -0,0 +1,135 @@
+import pytest
+from pathlib import Path
+
+from neurodamus.node import Node
+from tests.utils import record_compartment_report, write_ascii_report, read_ascii_report
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize("create_tmp_simulation_config_file", [
+    {
+        "simconfig_fixture": "ringtest_baseconfig",
+        "extra_config": {
+            "reports": {
+                "new_report": {
+                    "type": "compartment",
+                    "cells": "Mosaic",
+                    "variable_name": "wrong",
+                    "sections": "all",
+                    "dt": 0.1,
+                    "start_time": 0.0,
+                    "end_time": 40.0,
+                }
+            }
+        }
+    }
+], indirect=True)
+def test_report_config_error(create_tmp_simulation_config_file):
+    with pytest.raises(Exception):
+        n = Node(create_tmp_simulation_config_file)
+        n.load_targets()
+        n.create_cells()
+        n.enable_reports()
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize("create_tmp_simulation_config_file", [
+    {
+        "simconfig_fixture": "ringtest_baseconfig",
+        "extra_config": {
+            "reports": {
+                "new_report": {
+                    "type": "compartment",
+                    "cells": "Mosaic",
+                    "variable_name": "wrong",
+                    "sections": "all",
+                    "dt": 0.1,
+                    "start_time": 0.0,
+                    "end_time": 40.0,
+                    "enabled": False
+                }
+            }
+        }
+    }
+], indirect=True)
+def test_report_disabled(create_tmp_simulation_config_file):
+    n = Node(create_tmp_simulation_config_file)
+    n.load_targets()
+    n.create_cells()
+    n.enable_reports()
+    assert len(n.reports) == 0
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize("create_tmp_simulation_config_file", [
+    {
+        "simconfig_fixture": "ringtest_baseconfig",
+        "extra_config": {
+            "target_simulator": "NEURON",
+            "reports": {
+                "soma_v": {
+                    "type": "compartment",
+                    "cells": "Mosaic",
+                    "variable_name": "v",
+                    "sections": "soma",
+                    "dt": 10,
```

Comment:
any particular reason for `dt = 10`?

Reply:
No particular reason, I just want the report data to be shorter. Initially I planned to check the voltage data but gave up in the end as there is no spike.

Reply:
you can inject a current to force spikes

Reply:
Do you have an example of the stimulus?

Reply:
there are many in `tests/unit/test_current_injection.py`. I think the simplest one may be:

```
@pytest.mark.parametrize("create_tmp_simulation_config_file", [
    {
        "simconfig_fixture": "ringtest_baseconfig",
        "extra_config": {
            "inputs": {
                "Stimulus": {
                    "module": "linear",
                    "input_type": "current_clamp",
                    "delay": 5,
                    "duration": 1,
                    "node_set": "RingA",
                    "represents_physical_electrode": True,
                    "amp_start": 10
                }
            }
        }
    },
], indirect=True)
```

this is just 1 impulse at 10nA for 1ms after 5ms delay I think

Reply:
I feel like I am missinterpreting your question, let me know if you wanted something else

Reply:
yes, that's what I want. I will try to add it. Thanks!

Reply:
I've added the same stimulus as `test_current_injection` and check the soma voltage peak with the same expected result. Can you review again ?